tempest/src/test/kotlin/app/cash/tempest/LogicalDbTransactionTest.kt

```diff
@@ -190,9 +191,10 @@ class LogicalDbTransactionTest {
     // Introduce a race condition.
     musicTable.playlistInfo.save(playlistInfoV2)
 
-    assertFailsWith<TransactionCanceledException> {
+    val exception = assertThrows<AmazonServiceException> {
```

Comment:
Unfortunate consequence of a bug in the DynamoDb Local client where it throws a different exception than what would be thrown in a real-world scenario.